lib/improver/cli/ensemble_calibration.py

```diff
@@ -170,66 +170,142 @@ def main(argv=None):
              'coefficients to solve for.')
     args = parser.parse_args(args=argv)
 
+    # Load cubes
     current_forecast = load_cube(args.input_filepath)
     historic_forecast = load_cube(args.historic_filepath)
     truth = load_cube(args.truth_filepath)
+    # Process Cubes
+    result = process(current_forecast, historic_forecast, truth, args.units,
+                     args.distribution, args.predictor_of_mean,
+                     args.num_realizations, args.random_ordering,
+                     args.random_seed, args.ecc_bounds_warning,
+                     args.max_iterations)
+
+    # Save Cube
+    save_netcdf(result[0], args.output_filepath)
+    # If required, save the mean and variance.
+    if args.save_mean:
```

Comment:
The function has to return a tuple as there are move saving option here. Is this thee best way to do it?

Reply:
I would unpack the returned tuple on assignment:
```python
result, forecast_predictor, forecast_variance = process(...)
```

Reply:
thank you